angular-beans/src/main/java/angularBeans/util/CommonUtils.java

```diff
@@ -214,21 +212,43 @@ public static String obtainFieldNameFromAccessor(String getterName) {
 		return fieldName;
 	}
 	
-	public static boolean isNullOrEmpty(Object o){
-		//TODO gotta complete this dammy method
-		if (o == null){
+	// TODO use following methods into the code.
+	/**
+	 * check is the parameter is null or empty.
+	 * 
+	 * @param <T>
+	 *            class type of tested objects.
+	 * @param o
+	 *            parameter to check.
+	 * @return true is the parameter is null or empty, false otherwise.
+	 */
+	public static <T> Boolean isNullOrEmpty(final T o) {
+		if (o == null) {
 			return true;
 		}
-		if (o instanceof String){
-			return Strings.isNullOrEmpty((String) o);
+		if (o instanceof String) {
+			return "".equals(((String) o).trim());
+		}
+		if (o.getClass().isArray()) {
```

Comment:
But I want more your feedback about the duplicate callback enum class in two packages (after enum package elimination :( )